firmware/common2015/utils/logger/logger.cpp

```diff
@@ -5,18 +5,26 @@
 
 #include <mbed.h>
 #include <rtos.h>
+#include <cstdio>
+#include <errno.h>
 
 const char* LOG_LEVEL_STRING[] = {FOREACH_LEVEL(GENERATE_STRING)};
+const char* LOG_FILE_NAMES[] = {"/local/LOG1.TXT", "/local/LOG2.TXT"};
+
+const int maxLogSize = 500000;  // bytes
```

Comment:
The MBED only has 2 MB of space total, so I was trying to be a little conservative.
Overall, the startup sequence produces ~700 bytes worth of logs. And with my current setup, there are no periodic log calls.